Gemfile

```diff
@@ -28,8 +32,9 @@ gem 'oink'
 gem 'paperclip', '~> 5'
```

Comment:
I think we should make a note/todo here to switch to activestorage once rails version is upgraded

Reply:
Yes - have a look at the plan in Redmine and you will see that there's a series of upgrade plans coming up, and this is among them. I'm very eager to make that switch but you are right that it is blocked by the rails upgrade.